src/Orchard/Environment/Extensions/Models/ExtensionDescriptor.cs

```diff
@@ -12,6 +12,8 @@ public class ExtensionDescriptor {
         /// </summary>
         public string Id { get; set; }
 
+        public string VirtualPath { get { return Location + "/" + Id; } }

```

Comment:
Fair enough. The concatenation is still not right, even if this was the case before.